vmngclient/api/templates.py

```diff
@@ -242,14 +291,15 @@ def load_running(self, device: Device) -> None:
             device (Device): The device from which load config.
         """
         endpoint = f"/dataservice/template/config/running/{device.uuid}"
-        config = cast(dict, self.session.get_json(endpoint))
+        config = self.session.get_json(endpoint)
         self.config = CiscoConfParse(config['config'].splitlines())
+        logger.info(f"Template loaded from {device.hostname}.")
```

Comment:
```suggestion
        logger.debug(f"Template loaded from {device.hostname}.") # Maybe add template text for debugging?
```

Reply:
at the next PR, config logging to the logger will be added